python/ray/_private/worker.py

```diff
@@ -2620,10 +2643,57 @@ def get(
         return values
 
 
+@PublicAPI
```

Comment:
stabaility="alpha"?

Actually should we even annotate it as public API? Maybe we can do it later in the future?